auto_nag/scripts/missing_beta_status.py

```diff
@@ -13,7 +13,7 @@ def __init__(self):
         if not self.init_versions():
             return
 
-        self.status_central = utils.get_flag(
+        self.status_nightly = utils.get_flag(
```

Comment:
Nit: I'd keep the old variable name to minimize the diff, no need to change (central can be considered a synonym of nightly)

Reply:
I changed it to be more consistent with the comments and the descriptions. I did that in a separate commit to minimize the diff. I will revert the change.